nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow_schema.json

```diff
@@ -5,195 +5,237 @@
-            "fa_icon": "fas fa-fingerprint",
-            "help_text": "A custom name for the pipeline run. Unlike the core nextflow `-name` option with one hyphen this parameter can be reused multiple times, for example if using `-resume`. Passed through to steps such as MultiQC and used for things like report filenames and titles.",
-            "hidden": true
-        },
-        "email": {
-            "type": "string",
-            "description": "Email address for completion summary",
-            "fa_icon": "fas fa-envelope",
-            "help_text": "An email address to send a summary email to when the pipeline is completed.",
-            "pattern": "^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$"
-        },
-        "email_on_fail": {
-            "type": "string",
-            "description": "Email address for completion summary, only when pipeline fails",
-            "fa_icon": "fas fa-exclamation-triangle",
-            "pattern": "^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$",
-            "help_text": "An email address to send a summary email to when the pipeline is completed - ONLY sent if the pipeline does not exit successfully."
-        },
-        "plaintext_email": {
-            "type": "boolean",
-            "description": "Send plain-text email instead of HTML",
-            "fa_icon": "fas fa-remove-format",
-            "hidden": true,
-            "default": false
-        },
-        "multiqc_config": {
-            "type": "string",
-            "description": "Custom config file to supply to MultiQC",
-            "default": "",
-            "fa_icon": "fas fa-cog",
-            "hidden": true
-        },
-        "max_multiqc_email_size": {
-            "type": "string",
-            "description": "File size limit when attaching MultiQC reports to summary emails",
-            "default": "25 MB",
-            "fa_icon": "fas fa-file-upload",
-            "hidden": true
-        },
-        "publish_dir_mode": {
-            "type": "string",
-            "default": "copy",
-            "hidden": true,
-            "description": "Method used to save pipeline results to output directory",
-            "help_text": "The Nextflow `publishDir` option specifies which intermediate files should be saved to the output directory. This option tells the pipeline what method should be used to move these files. See https://www.nextflow.io/docs/latest/process.html#publishdir for details.",
-            "fa_icon": "fas fa-copy",
-            "enum": [
-                "symlink",
-                "rellink",
-                "link",
-                "copy",
-                "copyNoFollow",
-                "mov"
-            ]
-        },
-        "monochrome_logs": {
-            "type": "boolean",
-            "description": "Do not use coloured log outputs",
-            "fa_icon": "fas fa-palette",
-            "hidden": true,
-            "default": false
+        "Generic options": {
```

Comment:
Not sure about this name. Maybe _Input data_?

Reply:
I wasnt either but just thought I would create a group that contained more generic parameters because I had some that didnt fit in the others!

Changed: https://github.com/nf-core/tools/pull/593/commits/4c0ac910be308a549c7e0ee0ec55cce96cbcc414